ci/ray_ci/bisect/release_test_bisect.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -eux
+
+# set up the release and bisect infra environment
+bash release/gcloud_docker_login.sh release/aws2gce_iam.json 
+export PATH="$(pwd)/google-cloud-sdk/bin:$PATH"
+export BUILDKITE_MAX_RETRIES=0
+
+pip install -r release/requirements_buildkite.txt
+pip install -e release/
+
+
+# running bisect
+TEST_NAME=$(buildkite-agent meta-data get test-name)
+PASSING_COMMIT=$(buildkite-agent meta-data get passing-commit)
+FAILING_COMMIT=$(buildkite-agent meta-data get failing-commit)
+CONCURRENCY=$(buildkite-agent meta-data get concurrency)
+RUN_PER_COMMIT=$(buildkite-agent meta-data get run-per-commit)
+
+cd release 
+python ray_release/scripts/ray_bisect.py \
```

Comment:
soft nudge: consider make this bazel run? then you do not need to run `pip install` and do not need to care about which python version it has in the forge base container image.